packages/foam-vscode/src/test/test-utils-vscode.ts

```diff
@@ -13,7 +13,7 @@ import { randomString, wait } from './test-utils';
 Logger.setLevel('error');
 
 export const cleanWorkspace = async () => {
-  const files = await vscode.workspace.findFiles('**', '.vscode');
+  const files = await vscode.workspace.findFiles('**', '{.vscode,.keep}');
```

Comment:
At the end I decided against rimraf here because I was seeing obscure errors that could be related to syncing issues between the underlying fs and vscode's view of it. And, anyway, it wasn't that easy to remove folders using glob patterns.